### DISCUSSION TITLE
extract_table header not recognized with body when there is a blank row

### DISCUSSION THREAD

**Discussion by yonglee7015** [jsvine/pdfplumber#776] [Ask for help with specific PDFs]
header and body there is a blank, how to extract header and body together?
Fuction -- extract_table always ignore header, where do I need to adjust?

![table_header](https://user-images.githubusercontent.com/59268746/207529138-58cf0cf4-fb98-4686-8d79-86848c982083.PNG)

**Comment by yonglee7015** [comment]
Please close this discussion. I use extract_tables, then combine both tables with first find table's header as column header.

**Reply by samkit-jain** [reply]
You can also use `join_tolerance` to combine the close by edges.

**Reply by yonglee7015** [reply]
Thank you. But It is difficult to find the most appropriate value. I will try it later.
